packages/cypress-plugins/src/plugins/networkShim/index.js

```diff
@@ -37,12 +37,6 @@ module.exports = function networkShim(
             state = newState
             return state
         },
-        setNetworkShimMissingRequestStubs(missingRequestStubs) {
-            if (Array.isArray(missingRequestStubs)) {
-                state.missingRequestStubs = missingRequestStubs
-            }
-            return state
-        },
```

Comment:
This has become redundant due to the changes in `packages/cypress-commands/src/setups/enableNetworkShim/index.js`